sdk/openai/Azure.AI.OpenAI/README.md

```diff
@@ -39,7 +39,7 @@ Once you have the value of the endpoint string and subscription key, you can cre
 // Replace with your Azure OpenAI key
 string key = "YOUR_AZURE_OPENAI_KEY";
 string endpoint = "https://myaccount.openai.azure.com/";
-OpenAIClient client = new OpenAIClient(new Uri(endpoint), new AzureKeyCredential(key));
+OpenAIClient client = new OpenAIClient(new Uri(endpoint), "myModelDeploymentId", new AzureKeyCredential(key));
```

Comment:
This definitely does not and should not hold up the PR in any way, it's a thing for followup consideration: given our scenarios (embeddings, completion, likely chat completions) have specific and mutually exclusive model/deployment requirements, this will effectively lock an instantiated client into use with a single scenario, correct?

If so, I think we need to (after this PR, still!) consider either per-scenario deployment specification (e.g. as distinct fields on the client) or per-scenario *clients* (CompletionsClient, EmbeddingsClient, etc.).

Reply:
Agreed, I'd like feedback from @m-nash, @tg-msft, and @johanste, as well as the Speech stakeholders (you, @jpalvarezl, gerardo), but I think a shim scenario-specific client could definitely make sense and ease the cognitive burden on users.